toolkit/docs/quick_start/quickstart.md

```diff
@@ -0,0 +1,128 @@
+# Quick Start Guide
+  - [Install Prerequisites](#install-prerequisites)
+  - [Clone CBL-Mariner](#Clone-CBL-Mariner)
+  - [Build and Boot an Image](#build-and-boot-an-image)
+    - [VHDX and VHD Images](#vhdx-and-vhd-images)
+    - [ISO Image](#iso-image)
+  
+## **Install Prerequisites**
+Install prerequisites [here](../building/prerequisites.md).
+
+ 
+## **Clone CBL-Mariner**
+From a bash terminal window, clone the CBL-Mariner Repository and check-out a stable build.
+
+```bash
+# Clone the CBL-Mariner repo
+git clone git@github.com:microsoft/CBL-Mariner.git
+cd CBL-Mariner
+
+# Sync to the latest stable build 
+git checkout 1.0-stable
+
+```
+
+## **Build and Boot an Image**
+
+### **VHDX and VHD images**
+
+The following builds a bootable, VHDX or VHD CBL-Mariner image from precompiled RPMs in the CBL-Mariner package repository at https://packages.microsoft.com/cbl-mariner/1.0/prod/.
+
+No user account is provisioned by default.  To sign-in to these images, the sample meta-user-data.iso image must also be built and installed in your VM's CD drive.  The cloud-init service will detect the iso and provision a user account and password.  
+
+**Build a VHD or VHDX Image**
+```bash
+# Switch to the toolkit folder 
+cd toolkit
+
+# Build VHDX Image
+# Image is placed in ../out/images/core-efi
+sudo make image REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/core-efi.json
+
+# Build VHD Image
+# Image is placed in ../out/images/core-legacy
+sudo make image REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/core-legacy.json
+```
+
+**Build the cloud-init configuration image**
+```Bash
+# Build the cloud-init configuration image 
+# The output image is ../out/images/meta-user-data.iso
+sudo make meta-user-data
+
+```
+
+**Copy VHD(X) and ISO Images to Your VM Host Machine**
```

Comment:
I'd drop this. The step seems rather obvious and an empty section looks as if we forgot to add the content.

Reply:
I think it needs to be there, but I agree with the "empty section" look and feel.  Added something so it's not empty.